src/components/Panel/index.js

```diff
@@ -1,24 +1,77 @@
 import React from 'react'
 import Editor from '../Editor'
 import Result from '../Result'
+import storage, { mkStore } from '../../utils/storage'
 import './style.css'
 
 const wrapInTry = code => {
   return `try{ ${code} } catch (e) {e}`
 }
 
+const snippets = mkStore('snippet__')
+
+class SnippetSelector extends React.Component {
```

Comment:
I think it would be great to put this component in its own file, so we don't overload this file.

Reply:
Done, I knew you were going to say this :D